connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorUtils.java

```diff
@@ -98,19 +112,51 @@ static Pattern compilePatternList(String fields) {
         return compilePatternList(Arrays.asList(fields.split("\\W*,\\W*")));
     }
 
-    static void createCompactedTopic(String topicName, short partitions, short replicationFactor, Map<String, Object> adminProps) {
+    static void createCompactedTopic(String topicName, short partitions, short replicationFactor, ForwardingAdmin forwardingAdmin) {
         NewTopic topicDescription = TopicAdmin.defineTopic(topicName).
                 compacted().
                 partitions(partitions).
                 replicationFactor(replicationFactor).
                 build();
 
-        try (TopicAdmin admin = new TopicAdmin(adminProps)) {
-            admin.createTopics(topicDescription);
+        CreateTopicsOptions args = new CreateTopicsOptions().validateOnly(false);
+        try {
+            forwardingAdmin.createTopics(singleton(topicDescription), args).values().get(topicName).get();
+            log.info("Created topic", topicName);
+        } catch (InterruptedException e) {
+            Thread.interrupted();
+            throw new ConnectException("Interrupted while attempting to create/find topic '" + topicName + "'", e);
+        } catch (ExecutionException e) {
+            Throwable cause = e.getCause();
+            if (cause instanceof TopicExistsException) {
+                log.debug("Found existing topic '{}'", topicName);
```

Comment:
If we want a debug line, let's add some details to it, for example `Unable to create topic <> since it already exists`.

Reply:
updated the log message.